framework/taskmanager/TaskManager.py

```diff
@@ -443,6 +452,51 @@ def run_publishers(self, actions, facts, data_block=None):
                 self.logger.debug('run publisher %s %s'%(self.channel.publishers[action].name, data_block))
                 self.channel.publishers[action].worker.publish(data_block)
 
+
+    def _create_facts_dataframe(self, le_result):
```

Comment:
If this function moved to LogicEngine, why is it still here?

Reply:
Thats cause i forgot to push the updated file :(. Now done